unimod_mapper/version.txt

```diff
@@ -1 +1 @@
-0.4.2dev1
+0.4.4
```

Comment:
versions should be bumped in dev or master :)

Reply:
that's because I merged the master into this branch to have the actual features, since we had this discrepancy between dev and master ^^